src/test/unit/AllocationManagerUnit.t.sol

```diff
@@ -996,115 +1007,135 @@ contract AllocationManagerUnitTests_SlashOperator is AllocationManagerUnitTests
+
+        _registerForOperatorSet(defaultOperator, operatorSet);
+        _registerForOperatorSet(defaultOperator, operatorSet2);
+
+        cheats.prank(defaultOperator);
+        allocationManager.modifyAllocations(allocateParams);
+        cheats.roll(block.number + DEFAULT_OPERATOR_ALLOCATION_DELAY);
+
+        // Get slashable shares for each operatorSet
+        address[] memory operatorArray = new address[](1);
+        operatorArray[0] = defaultOperator;
+        // (, uint256[][] memory slashableSharesOpset1_preSlash) = allocationManager
+        //     .getMinDelegatedAndSlashableOperatorSharesBefore(
+        //     OperatorSet(defaultAVS, 1), operatorArray, defaultStrategies, uint32(block.number + 1)
+        // );
+        // (, uint256[][] memory slashableSharesOpset2_preSlash) = allocationManager
+        //     .getMinDelegatedAndSlashableOperatorSharesBefore(
+        //     OperatorSet(defaultAVS, 2), operatorArray, defaultStrategies, uint32(block.number + 1)
+        // );
+        // assertEq(40e18, slashableSharesOpset1_preSlash[0][0], "slashableShares of opSet_1 should be 40e18");
+        // assertEq(40e18, slashableSharesOpset2_preSlash[0][0], "slashableShares of opSet_2 should be 40e18");
+        uint256 maxMagnitude = allocationManager.getMaxMagnitudes(defaultOperator, defaultStrategies)[0];
+        uint256 opSet2PortionOfMaxMagnitude = uint256(magnitudeToAllocate) * WAD / maxMagnitude;
+
+        // Slash operator on operatorSet1 for 50%
+        SlashingParams memory slashingParams = SlashingParams({
+            operator: defaultOperator,
+            operatorSetId: allocateParams[0].operatorSet.id,
+            wadToSlash: 5e17,
+            description: "test"
+        });
+
+        _checkSlashEvents({
+            operator: defaultOperator,
+            operatorSet: operatorSet,
+            strategies: defaultStrategies,
+            wadToSlash: uint256(5e17).toArrayU256(),
+            description: "test"
+        });
+
+        // Slash Operator
+        cheats.prank(defaultAVS);
+        allocationManager.slashOperator(slashingParams);
+
+        // Operator should now have 80e18 shares, since half of 40e18 was slashed
+        delegationManagerMock.setOperatorShares(defaultOperator, strategyMock, 80e18);
+
+        // // Check storage
+        // (, uint256[][] memory slashableSharesOpset1_postSlash) = allocationManager
+        //     .getMinDelegatedAndSlashableOperatorSharesBefore(
+        //     OperatorSet(defaultAVS, 1), operatorArray, defaultStrategies, uint32(block.number + 1)
+        // );
+        // (, uint256[][] memory slashableSharesOpset2_postSlash) = allocationManager
+        //     .getMinDelegatedAndSlashableOperatorSharesBefore(
+        //     OperatorSet(defaultAVS, 2), operatorArray, defaultStrategies, uint32(block.number + 1)
+        // );
+
+        // assertEq(20e18, slashableSharesOpset1_postSlash[0][0], "slashableShares of opSet_1 should be 20e18");
+        // assertEq(
```

Comment:
Are we testing this in a separate func?

Reply:
The fn was removed, need to find where this state lives now.